app/models/UpdateCaseError.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.libs.json.{Json, Writes}
+
+sealed trait UpdateCaseError extends Product with Serializable
+
+object UpdateCaseError {
+  case object InvalidCaseId extends UpdateCaseError
+  case object CaseAlreadyClosed extends UpdateCaseError
+  final case class UnexpectedError(message: String, status: Int) extends UpdateCaseError
+
+  def fromEisError(error: EisError): UpdateCaseError = {
+    error match {
+      case EisError.BackendError(_, _, "03- Invalid Case ID") => UpdateCaseError.InvalidCaseId
```

Comment:
there is a space missing before the hyphen, but it's been like that in the docs :man_shrugging: